### PR TITLE
Saeed/add logger

### LINKED ISSUE #9: adding logs
I want to add logger file to proj do you agree
CC @Alirexaa

### PR DESCRIPTION
**Pull request by SaeedSafi1999**

### CODE REVIEW

File: Log/logger.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+[INFO] at:2024-08-15T16:53:40.765750200+03:30 =>: application starting...
```
Comment (Alirexaa) [inline Log/logger.txt]: Why we having this file in history?

Comment (SaeedSafi1999) [reply Log/logger.txt]: this is test for instance of now date let me fix that

Comment (SaeedSafi1999) [reply Log/logger.txt]: commit new one

File: Cargo.toml [hunk 2]
```diff
@@ -5,6 +5,7 @@ edition = "2021"
 build = "build.rs"
 
 [dependencies]
+chrono = "0.4"
```
Comment (Alirexaa) [inline Cargo.toml:8]: Do we need this?

File: build.rs [hunk 3]
```diff
@@ -25,10 +26,17 @@ fn main() {
     nssm_dest.push(".."); 
     nssm_dest.push(".."); 
     nssm_dest.push("nssm");
+
+    let mut logger_dest = PathBuf::from(out_dir.clone());
+    logger_dest.push(".."); 
+    logger_dest.push(".."); 
+    logger_dest.push(".."); 
+    logger_dest.push("Logs");
```
Comment (Alirexaa) [inline build.rs]: why we need to move log file to the build?

File: Cargo.toml (hunk 2 above)
Comment (SaeedSafi1999) [reply Cargo.toml:8]: yes this is format for date

File: build.rs (hunk 3 above)
Comment (SaeedSafi1999) [reply build.rs]: i fix this now

**Comment by SaeedSafi1999** [comment]
@Alirexaa

**Comment by Alirexaa** [comment]
LGTM

**Review by Alirexaa — APPROVED** [review]

**Pull request merged**